.env.example

```diff
@@ -2,7 +2,7 @@
 HTTP_PORT=8080
 # EXTERNAL_HTTP_PORT=8089
 DB_PORT=8123
-DB_HOST=http://clickhouse
+DB_HOST=http://localhost
```

Comment:
It works for "via Node" run only. We have the separate section for it here - https://github.com/lidofinance/ethereum-validators-monitoring?tab=readme-ov-file#run-via-node
Let's keep `http://clickhouse` because docker is the main approach to run application